ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -477,6 +482,21 @@ class ActionBar extends React.PureComponent {
                   </a>
                 </li>
               ))}
+            <li>
+              <Button
+                id="find-job-btn"
+                title="Find this job instance"
+                className="actionbar-nav-btn btn icon-blue bg-transparent border-0"
+                onClick={() =>
+                  findJobInstance(jobLogUrls[0] && jobLogUrls[0].job_id, true)
```

Comment:
This is great.  And the new spacing fixes things nicely, I think.  Some day we may want to add a "pulse" of some kind.  Like have it scale up to 2.0, then back to 1.3 when you hit this button.  But don't have to do that now.